packages/dashboard/src/components/common/AnalyticsToggle.tsx

```diff
@@ -0,0 +1,26 @@
+import { Switch } from '@headlessui/react';
+import { useState } from 'react';
+
+export default function AnalyticsToggle() {
+  const [enabled, setEnabled] = useState(false);
+
+  return (
+    <Switch.Group>
+      <div className="flex justify-center mt-4">
+        <span className="pr-3 float-left">Disabled</span>
+        <Switch
+          checked={enabled}
+          onChange={setEnabled}
+          className={`${enabled ? 'bg-truffle-blue' : 'bg-truffle-brown'
+            } relative inline-flex items-center h-6 rounded-full w-11 transition-colors focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-truffle-blue`}
+        >
+          <span
```

Comment:
Accessibility audit:

This Switch element should contain a ` <span className="sr-only">{{ << PUT WHAT ENABLING THIS ENABLES HERE >> }}</span>` element. It might contain something like "Enable tracking". The value here may be used by screen readers (`sr-only` cl;ass here means "screen readers only").

Reply:
Ah! You know, I read up on accessibility with the popover component, and completely missed covering the switch element. This makes sense. Added -- is this what you meant? I checked in the developer tools but don't actually know how to make sure an screen reader will catch this!

Reply:
I'm just starting to get into the accessibility stuff myself, and I'm sure there is so much else we are doing wrong all over the place. I haven't tried it out on my mac yet, but I think you can enable "VoiceOver" in your system settings and try it out by navigating with the Tab key, or by hovering with the mouse.

> This switch enables anonymous analytics tracking

Something else I just thought of: the toggle doesn't only enable tracking, it also disables it. Something like "Enable anonymized tracking" and "Disable anonymized tracking" is probably enough. ¯\_(ツ)_/¯ Anything is better than nothing, of course!